models/intel/text-recognition-0013/description/text-recognition-0013.md

```diff
@@ -0,0 +1,53 @@
+# text-recognition-0013
+
+## Use Case and High-Level Description
+
+This is a network for text recognition scenario. It consists of VGG16-like backbone and bidirectional LSTM encoder-decoder.
+The network is able to recognize case-insensitive alphanumeric text (36 unique symbols).
+
+## Example
+
+![](./openvino.jpg) -> openvino
+
+## Specification
+
+| Metric                                         | Value              |
+| ---------------------------------------------- | ------------------ |
+| Accuracy on the alphanumeric subset of ICDAR13 | 0.8739             |
+| Text location requirements                     | Tight aligned crop |
+| GFlops                                         | 1.488              |
+| MParams                                        | 5.567              |
+| Source framework                               | PyTorch            |
+
+## Inputs
+Input tensor is `imgs`.
+Shape: `1, 1, 32, 120` - An input image in the format `B, C, H, W`,
+where:
+  - B - batch size
+  - C - number of channels
+  - H - image height
+  - W - image width
+
+Note that the source image should be tight aligned crop with detected text converted to grayscale.
+
+## Outputs
+The net outputs a blob `logits` with the shape `30, 1, 37` in the format `W, B, L`,
```

Comment:
Given demo has `-tr_o_blb_nm`, I think there should be multiple outputs listed.

Reply:
All the other outputs are meaningless: they are created after model optimizer step and should not exist in the model. In other words, these are handing nodes.

Reply:
`input.yml` lists "targets".

Reply:
Oh, I've forgotten about this output since it is not used in accuracy checker nor in demo. Added info bout this output.

Reply:
Are you sure it has rank 3? For me IE prints 2.